android/app/src/main/res/layout/search_result_type_header_format.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.v7.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:elevation="4dp"
+    app:cardCornerRadius="0dp"
+    android:layout_height="wrap_content">
+
+    <LinearLayout
```

Comment:
Why the need of separate header? If you nevertheless need it, please use TextView as direct child of CardView instead

There is no need of wrapping it in LinearLayout